catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventHandlerDelegate.java

```diff
@@ -137,19 +147,12 @@ public Metacard read(InputStream inputStream) throws CatalogTransformerException
         for (SaxEventHandler eventHandler : eventHandlers) {
             List<Attribute> attributes = eventHandler.getAttributes();
             for (Attribute attribute : attributes) {
-                Attribute tmpAttr;
 
                 /*
-                 * If metacard already has values in the attribute, put them together into a multivalued list,
+                 * If metacard already has values in the attribute, skip the attribute,
                  * instead of simply overwriting the existing values.
                  */
-                if ((tmpAttr = metacard.getAttribute(attribute.getName())) != null) {
```

Comment:
Yes. This was discussed because some attributes were being multivalued that shouldn't have been.  The change was made so that the first value set, wins. If this proves to be an issue we can revisit it.